config/locales/en-GB.yml

```diff
@@ -529,7 +529,7 @@ en-GB:
         welcome_title: Welcome to the Open Food Network!
         welcome_text: You have successfully created a
         next_step: Next step
-        choose_starting_point: 'Choose your starting point:'
+        choose_starting_point: 'Choose your package:'
```

Comment:
One thing you couldn't know, is that we change only `en.yml` manually. That is our source locale that gets translated with Transifex. The translations from Transifex get then merged into our code automatically. So if you could revert you changes on all other locales, that would be great. Otherwise we will face merge conflicts.